experimental/addons/koordinator/README.md

```diff
@@ -0,0 +1,117 @@
+# koordinator
+
+Koordinator is a QoS based scheduling system for hybrid workloads orchestration on Kubernetes. It aims to improve the runtime efficiency and reliability of both latency sensitive workloads and batch jobs, simplify the complexity of resource-related configuration tuning, and increase pod deployment density to improve resource utilizations.
+
+## Install
+
+Add experimental registry
+```
+vela addon registry add experimental --type=helm --endpoint=https://addons.kubevela.net/experimental/
+```
+
+Enable this addon
+```
+vela addon enable koordinator
+```
+
+```shell
+$ vela ls -A | grep koordinator
+prod            koordinator-reservation-sample  koordinator-reservation                 koordinator-reservation                                 running healthy
+vela-system     addon-koordinator               koordinator                             helm                                                    running healthy Fetch repository successfully, Create helm release
+```
+
+Disable this addon
+```
+vela addon disable koordinator
+```
+
+## Use
+### koordinator
```

Comment:
This scenario mainly uses Koordinator's oversold resources and QoS mechanism, maybe changed to `Use Koordinator colocated workloads`?

In addition, have you considered adding a few distinctive scenes? For example [Fine-grained CPU orchestration](https://koordinator.sh/docs/user-manuals/fine-grained-cpu-orchestration) and [load-aware scheduling](https://koordinator.sh/docs/user-manuals/load-aware-scheduling).

Reply:
Not yet, But very soon I'll have a look on a few distinctive scenes and get back to you.

Reply:
@wonderflow @eahydra  @wangyikewxgm  Should I add a few more distinctive scenes in README? or I should keep README brief and simple?

Reply:
IMO we should add some more featured scenes in the README.  WDYT?